### PR TITLE
Basic .CSV file exporter.

### PR DESCRIPTION
**Pull request by ShaneKent**
TODO:
 - Possible different file type to export with color, etc.
 - Filtering (once it's complete) will need to effect the export.

Issues:
 - Sorting of the timeline doesn't effect the export.
 - Opening the .CSV file isn't a streamlined process. Possibly export to .XCL or .PDF format instead.

### CODE REVIEW

**Comment by ShaneKent** [comment]
Tied to Issue #49

File: winlogtimeline/ui/ui.py [hunk 1]
```diff
@@ -521,14 +528,15 @@ def import_button_function(self, event=None):
...
         :param event:
         :return:
         """
-        self.master.status_bar.status.config(text='"Format" button pressed.')
-        return
+        print(self.master.current_project)
+        wizard = ExportWindow(self, self.master.current_project)
+        wizard.grab_set()
 
     def __enable__(self):
```
Comment (carterv) [inline winlogtimeline/ui/ui.py]: Please add the export menu option to the __enable__ and __disable__ of the menu. The feature doesn't work on macs, but it still looks cleaner than a status bar message on Windows.

File: winlogtimeline/ui/export_timeline.py [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+            for record in records:
+                row = []
+
+                if columns["Timestamp (UTC)"]:
+                    row.append(record.timestamp_utc)
+                if columns["Event ID"]:
+                    row.append(record.event_id)
+                if columns["Description"]:
+                    row.append(record.decription)
```
Comment (carterv) [inline winlogtimeline/ui/export_timeline.py]: Typo: decription -> description. 

Additionally, you might want to look into itertools.compress here. You could turn this giant if block into a line or two.

File: winlogtimeline/ui/export_timeline.py [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        if self.delimiter_string.get() == "Comma":
+            delimiter = ","
+        elif self.delimiter_string.get() == "Space":
+            delimiter = " "
+        elif self.delimiter_string.get() == "Tab":
+            delimiter = "\t"
+
+        from csv import writer
+
+        with open(self.current_project.get_path() + '/{}.csv'.format(self.file_path.get()), 'w') as csvfile:
```
Comment (carterv) [inline winlogtimeline/ui/export_timeline.py]: Shouldn't the user have some input on where the file is exported to?

Comment (ShaneKent) [reply winlogtimeline/ui/export_timeline.py]: I figured no. I believe it makes sense to have the file exported to the project directory - it's at least a good use of the repository. @carterv Thoughts?

File: winlogtimeline/ui/ui.py (hunk 1 above)
Comment (ShaneKent) [reply winlogtimeline/ui/ui.py]: Ah good note. I didn't see that. That's been fixed.

**Comment by ShaneKent** [comment]
@carterv All comments have been addressed. Still am not allowing the user to choose the output directory of the exported timeline.

File: winlogtimeline/ui/export_timeline.py (hunk 3 above)
Comment (zjmonroe23) [reply winlogtimeline/ui/export_timeline.py]: Can we set that as default but allow them to change locations if they desire?

**Comment by ShaneKent** [comment]
@zjmonroe23 @carterv Thanks for accepting. I've added a commit to this request that allows users to change the location of the export. The default location is the current project directory. I will be merging now.

**Pull request merged**